GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3098,7 +3105,7 @@ private void undoLastCommitToolStripMenuItem_Click(object sender, EventArgs e)
             {
                 var args = GitCommandHelpers.ResetCmd(ResetMode.Soft, "HEAD~1");
                 Module.RunGitCmd(args);
-                RefreshRevisions();
+                refreshToolStripMenuItem.PerformClick();
```

Comment:
@RussKie Usually disallows usage like this, call a new method instead.

Reply:
Thank you
I don't like when event handlers are invoked directly, e.g. `refreshToolStripMenuItem_Click(null, null)`.
I don't have a problem with invoking a method on an object.